docs/examples/config/cas.php

```diff
@@ -0,0 +1,12 @@
+<?php
+$cas_setup = array(
+    'host'  =>  'localhost',
```

Comment:
i would prefer if you just return the array and in callee use `$cas_setup = require 'config.php';`